conf.py

```diff
@@ -4,10 +4,8 @@
 import re
 import sys
 
-sys.path.insert(0, os.path.abspath('_ext'))
```

Comment:
I think this either needs to be kept (but changed to `_extensions`), or the `Makefile` changed to also use `poetry`? (I think we'd sort-of agreed on the latter?)

Reply:
I'll fix up the makefile to use poetry over sphinx-build. This way we can at least guarantee everyone is using the same environment and the build is slightly more hermetic 🙂